File: 03--code/zz--archive/misc_code/histogram.R
# Navigate to directory with '\n'-separated list of VAFs
getwd()
setwd("/Users/leo/Desktop/temp/isec/m33_m22--isec--m33_m17")

# import CSV file as a variable
csv_file = read.csv(file = "0002_AF.csv", header = FALSE, sep = "\n")

# make histogram
hist(csv_file[,1], breaks = seq(0,100,10), main = "m22_T_het -isec- m17_T_het Uniques", xlab = "Allele Frequency")

File: 03--code/zz--archive/misc_code/histogram.py
import pandas as pd
import matplotlib.pyplot as plt

# use a newline-separated text file with AF values
in_text_file = '/Users/leo/Desktop/temp/m33_N_MEF_m17_T_het.AF.txt'

allele_freq = pd.read_csv(in_text_file, delimiter='\n')

allele_freq.hist(bins=10)
plt.xlabel('Allele Frequency')
plt.ylabel('Number of Calls')
plt.title('AF: m33_N_MEF v. m17_T_het')
plt.grid(False)
plt.xlim(left=0, right=100)
plt.savefig('/Users/leo/Desktop/image.png')
